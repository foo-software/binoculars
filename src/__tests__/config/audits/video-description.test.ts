import auditVideoDescription from '../../../config/audits/video-description';

describe('video-description audit', () => {
  it('should match snapshot', () => {
    expect(auditVideoDescription('en')).toMatchSnapshot();
  });
});
