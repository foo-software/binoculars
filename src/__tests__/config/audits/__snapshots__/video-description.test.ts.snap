// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`video-description audit should match snapshot 1`] = `
Object {
  "description": "Audio descriptions provide relevant information for videos that dialogue cannot, such as facial expressions and scenes. [Learn more](https://web.dev/video-description/).",
  "title": "\`<video>\` elements contain a \`<track>\` element with \`[kind=\\"description\\"]\`",
}
`;
